src/App.tsx

```diff
@@ -38,58 +39,120 @@ export function getReorderedGoods(
   // eslint-disable-next-line no-console
   console.log(sortType, isReversed);
 
-  return visibleGoods;
+  visibleGoods.sort((goods1, goods2) => {
+    switch (sortType) {
+      case SortType.ALPABET:
+        return goods1.localeCompare(goods2);
+
+      case SortType.LENGTH:
+        return goods1.length - goods2.length;
+
+      default:
+        return 0;
```

Comment:
return visibleGoods or null instead of return 0

Reply:
Nope, `0` is fine here, since it's `sort` method's callback, that should return number.